src/features/upload-file/adapters/aws-adapter.ts

```diff
@@ -0,0 +1,67 @@
+import { S3 } from 'aws-sdk'
+import BaseAdapter from './base-adapter'
+
+/**
+ * AWS Credentials which can be set for S3 file upload.
+ * If not given 'aws-sdk' will try to fetch them from
+ * environmental variables.
+ * @memberof module:@admin-bro/upload
+ */
+export type AWSOptions = {
```

Comment:
I've changed Interfaces to Types because better-docs generates entire Page for each interface where Type can be included on the page where there is a module documentation